rust/feature-flags/src/api/test_endpoint.rs

```diff
@@ -0,0 +1,118 @@
+use axum::{
```

Comment:
this file is for testing request traffic without hitting any of the services; not helpful for testing evaluation, but helpful for testing that the various types of requests that we can send are handled correctly.